frontend/catalyst/device/qjit_device.py

```diff
@@ -202,70 +197,84 @@ def extract_backend_info(
                 device.target_device._s3_folder  # pylint: disable=protected-access
             )
 
-    for k, v in capabilities.options.items():
-        if hasattr(device, v) and not k in device_kwargs:
-            device_kwargs[k] = getattr(device, v)
+    for k, v in getattr(device, "device_kwargs", {}).items():
+        if k in device_kwargs:
+            device_kwargs[k] = v
 
     return BackendInfo(dname, device_name, device_lpath, device_kwargs)
 
 
+def intersect_operations(
+    a: Dict[str, OperatorProperties], b: Dict[str, OperatorProperties]
+) -> Dict[str, OperatorProperties]:
+    """Intersects two sets of operator properties"""
+    return {k: a[k] & b[k] for k in (a.keys() & b.keys())}
+
+
+def intersect_mps(a: dict[str, list], b: dict[str, list]) -> dict[str, list]:
+    """Intersects two sets of measurement processes"""
+    return {k: list(set(a[k]) | set(b[k])) for k in (a.keys() & b.keys())}
```

Comment:
This is a little bit abstract. Why is the intersection operation for operations defined as __and__ but the intersection operation for MPS defined as an __or__? Maybe a link or small explanation in the docstring here can help.

Reply:
Done